src/components/axes/categoryAxis.ts

```diff
@@ -61,17 +59,15 @@ export module Axes {
     }
 
     /**
-     * Gets the tick label angle
-     * @returns {number} the tick label angle
+     * Gets the tick label angle in degrees.
      */
     public tickLabelAngle(): number;
     /**
-     * Sets the angle for the tick labels. Right now vertical-left (-90), horizontal (0), and vertical-right (90) are the only options.
-     * @param {number} angle The angle for the ticks
-     * @returns {Category} The calling Category Axis.
+     * Sets the tick label angle in degrees.
+     * Right now only vertical-left (-90), horizontal (0), and vertical-right (90) are supported.
```

Comment:
I'm not sure if I can interpret "vertical-left" / "horizontal" / "vertical-right".  Maybe just have the numbers?